### PR TITLE
#11: Load apply data from firebase into form

### LINKED ISSUE #11: Load existing apply data from firebase on login
Website should populate form with latest form submissions.

### PR DESCRIPTION
**Pull request by themichaelyang**
Fixes #11, loading the logged in user's firebase form data into the ApplyPage form.

Also adds:
- babel fixes
- webpack sourcemaps

### CODE REVIEW

File: src/js/modules/types.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@ import {User} from "firebase";
 
 export interface ReduxState {
   core: CoreState
+  formData: FormData;
+  loading: boolean;
```
Comment (NicholasLYang) [inline src/js/modules/types.ts]: I don’t think this is where you want to put it

File: webpack.config.js [hunk 2]
```diff
@@ -2,11 +2,12 @@ const path = require("path");
 
 module.exports = {
   mode: "development",
-  entry: "./src/js/clientApp.js",
+  entry: "./src/js/clientApp.jsx",
```
Comment (NicholasLYang) [inline webpack.config.js]: Any particular reason for this? Just wondering

File: src/js/clientApp.jsx [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import "@babel/polyfill";
```
[the remark this answers is not in the document]
Comment (themichaelyang) [reply src/js/clientApp.jsx]: Contains browser polyfills for babel features. I included so `async/await` syntax works as expected.

File: webpack.config.js (hunk 2 above)
Comment (themichaelyang) [reply webpack.config.js]: just really for my syntax highlighting

File: src/js/clientApp.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (NicholasLYang) [reply src/js/clientApp.jsx]: Cool

**Pull request merged**